app/forms/employment/EmployerNameForm.scala

```diff
@@ -27,15 +27,18 @@ object EmployerNameForm {
 
   val employerName: String = "name"
   val charLimit: Int = 74
+  val regex: String = "^[0-9a-zA-Z\\{À-˿’}\\- _&`():.'^]{1,74}$"
```

Comment:
I'm not sure your employer name input accepts \

Reply:
I see the problem

Reply:
first \\ changed to \\\\ because scala needs extra escapes